app/src/main/java/com/kylecombes/micrepeater/ui/main/VoiceAmplifierFragment.java

```diff
@@ -37,6 +46,18 @@ static VoiceAmplifierFragment newInstance() {
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         pageViewModel = new ViewModelProvider(Objects.requireNonNull(getActivity())).get(AppStateViewModel.class);
+        String output = getActivity().getSharedPreferences("PREFERENCE", Context.MODE_PRIVATE)
+                .getString("audioOutput", "voice");
+        Log.v("out", output);
+        if (output.equals("voice")){
+            current_output = 0;
+        } else if (output.equals("alarm")){
+            current_output = 1;
+        } else {
+            current_output = 2;
+        }
```

Comment:
One thing you might consider using here for `output` is an enum.

```java
public enum AudioMode {
   ALARM,
   MUSIC,
   VOICE,
}
private AudioMode mAudioMode = AudioMode.MUSIC;
```
That way you're never at risk of typing in the wrong string or having to remember what the different strings are (you can just type `AudioMode.` and your IDE will tell you what the valid options are).

Reply:
Oh wait, you can't do that because `output` is loaded from `SharedPreferences`. Maybe call it `preferredOutput` then? It'd be nice if there were some indication as to where it came from.

Reply:
Also I feel like all of this could go in a nice little function that is called from `onCreateView` and the return value is used to set the position of the spinner. I don't think we really need to keep this in a class variable, do we?